src/components/bounty-setup/BountySetup.svelte

```diff
@@ -37,11 +38,20 @@
 				bountySetupTabClicked={onTabClick}
 			/>
 		</div>
-		<div class="grid bg-white border-2 border-gray-300 p-4">
-			<div class="rounded-t-md bg-gray-100 overflow-clip">
-				<BountyCreation />
+
+		{#if activeTab == TABS[0]}
```

Comment:
```suggestion
		{#if activeTab === TABS[0]}
```

We probably should add a lint rule to only allow strict equality.